components/camel-cxf/camel-cxf-soap/src/main/docs/cxf-component.adoc

```diff
@@ -749,11 +825,44 @@ is removed in `RAW` mode), you have to configure
 `LoggingOutInterceptor` to be run during the `WRITE` phase. The
 following is an example.
 
+[tabs]
+====
+Java::
++
+[source,java]
+----
+import java.util.List;
+import org.apache.camel.builder.RouteBuilder;
+import org.apache.camel.component.cxf.common.DataFormat;
+import org.apache.camel.component.cxf.jaxws.CxfEndpoint;
+import org.apache.cxf.interceptor.LoggingOutInterceptor;
+import org.apache.cxf.phase.Phase;
+import jakarta.enterprise.context.SessionScoped;
+import jakarta.enterprise.inject.Produces;
+import jakarta.inject.Named;
+
+...
+
+@Produces
```

Comment:
@ppalaga Are those annotations Quarkus specific ? Or it is also for Spring Boot ?

Reply:
I am not sure.

Reply:
I labeled the snippets with Quarkus/Spring in 0d0f9456e7b9facff69e840b99e49eca9671f0a2